### ISSUE TITLE
Iteration steps

### ISSUE THREAD

**Issue by mdyao** [sefibk/KernelGAN#14]
In the paper, "The GAN trains for 3,000 iterations, alternating single optimization steps of G and D, with the
ADAM optimizer (β1 = 0:5; β2 = 0:999). Learning rate is 2e−4, decaying ×0.1 every 750 iters."
why choose iteration to be 3000, what if more or fewer iterations?

I tried different iterations and get different kernel results. Please let me know your thoughts.

**Comment by sefibk** [comment]
After 3k iteration we observed that the GAN's kernel estimation is stable.
In the majority of cases it converged much earlier but since it re-trains for every image - there is no single correct number.
And since it re-trains for every image, there is some variance in the kernel estimation even when trained on the same image - so your observation makes sense.
That is the main limitation of the method of re-training for every image
